sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -206,6 +206,86 @@ export type AcsSmsReceivedEventData = AcsSmsEventBase & {
     receivedTimestamp: string;
 };
 
+// @public
+export interface AcsUserDisconnectedEventData {
+    userCommunicationIdentifier: CommunicationIdentifierModel;
+}
+
+// @public
+export interface ApiManagementApiCreatedEventData {
```

Comment:
maybe there's no handy way of doing this, but I wonder if we could tweak the codegen here to look like

```ts
export interface ApiManagementEventData {
   resourceUri: string;
}

export type ApiManagementApiCreatedEventData  = ApiManagementEventData;
export type ApiManagementApiDeletedEventData  = ApiManagementEventData;
```

Reply:
Agree this would be more idiomatic - will look into this on the side to see if there's a way to we can do this sort of generation in these cases (or if there's an easy set of auto-rest transforms we can put in our MD to have the existing generator give something more like this shape).